src/domain/landingPage/landingPageMeta/__tests__/LandingPageMeta.test.tsx

```diff
@@ -1,13 +1,15 @@
-import { render } from '@testing-library/react';
 import * as React from 'react';
 
 import { LandingPageFieldsFragment } from '../../../../generated/graphql';
 import { fakeBanner, fakeLandingPage } from '../../../../test/mockDataUtils';
-import { actWait } from '../../../../test/testUtils';
+import { actWait, render } from '../../../../test/testUtils';
 import LandingPageMeta from '../LandingPageMeta';
 
 const landingPageTitle = 'Landing page title';
-const landingPageDescription = 'Landing page description';
+// this value is localized.
+const landingPageDescription =
```

Comment:
This test could be moved to PageWrapper test accordingly

Reply:
see the comment about getDescription()